ska_conda/ska_builder.py

```diff
@@ -9,34 +9,56 @@
 
 class SkaBuilder(object):
 
-    def __init__(self, ska_root=None):
-        if ska_root is None:
-            ska_root = "/data/acis/ska3_pkg/"
-        self.ska_root = ska_root
-        self.ska_build_dir = os.path.join(self.ska_root, "builds")
-        self.ska_src_dir = os.path.join(self.ska_root, "src")
-        os.environ["SKA_TOP_SRC_DIR"] = self.ska_src_dir
+    def __init__(self, build_root='.', user='sot', git_repo_path='git@github.com:{user}/{name}.git'):
+        self.user = user
+        self.git_repo_path = git_repo_path
+        self.build_dir = os.path.abspath(os.path.join(build_root, "builds"))
+        self.src_dir = os.path.abspath(os.path.join(build_root, "src"))
```

Comment:
I don't know if there would be any other issues using abspath when trying to work somewhat relatively (using a build root explicitly defined as "." or something else relative), but this seems to work.

Reply:
Using abspath tends to make logging output cruftier than required, but otherwise should almost always be OK.